Tasks/AzureRmWebAppDeployment/postDeploymentScript/deleteLogFile.cmd

```diff
@@ -0,0 +1,22 @@
+@echo off
+
+DEL "mainCmdFile_%1.cmd" 
+
+DEL "kuduPostDeploymentScript_%1.cmd"
+
+:: Do not delete file if the execution is not completed within time range
+:: This can help in retrieving the logs and script result
+
+if exist "script_result_%1.txt" (
+
+    DEL "stdout_%1.txt"
+
+    DEL "stderr_%1.txt"
```

Comment:
If the script throws error that file not found and we provide it in debug logs, as deployment is not affected by it.
If file not found during getFileContent, then we throw exception.